cachi2/interface/cli.py

```diff
@@ -171,10 +172,7 @@ def fetch_deps(
     \b
     # gomod package in a subpath of the source directory (./my-repo/subpath)
     cachi2 fetch-deps --source ./my-repo '{
-        "type": "gomod",
-        "path": "subpath"
-    }'
-
+        "type": "gomod",backup_sources_and_resolve_packages
```

Comment:
Ah, know that feeling, it's been a while since I had that kind of mouse, but mine never had the option to paste as the rapid scrolling feature was triggered by clicking the wheel....anyway....